invenio_search_ui/assets/semantic-ui/js/invenio_search_ui/util.js

```diff
@@ -15,14 +15,22 @@ import _camelCase from "lodash/camelCase";
 export function createSearchAppInit(
   defaultComponents,
   autoInit = true,
-  autoInitDataAttr = "invenio-search-config"
+  autoInitDataAttr = "invenio-search-config",
+  allowMultipleApps = false
```

Comment:
What about shorter, with some doc string documenting this API interface/params?
```suggestion
  multi = false
```